dotcom-rendering/src/model/badges.ts

```diff
@@ -0,0 +1,238 @@
+import type { FEBadgeType, FESpecialBadgeType } from '../types/badge';
+
+/**
+ * Badges migrated from [frontend implementation](https://github.com/guardian/frontend/blob/c52cc1d35cec5a0ec95075f5e2079595d26dc35f/common/app/model/Badges.scala)
+ * @todo There are much nicer ways to store this kind of data
```

Comment:
do we plant to address this in this PR? we could use parameter store! this would also make it easier to add badges in future

Reply:
can also do it in a separate PR as this is getting quite big :D